library/azure_rm_keyvaultkey.py

```diff
@@ -195,6 +178,35 @@ def auth_callback(server, resource, scope):
 
         return self.results
 
+    def get_keyVaultClient(self):
```

Comment:
naming convention get_keyvault_client

Reply:
keyvaultClient is class name. anyway, i can update it